examples/wallet-server-hooks/next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
```

Comment:
just double-checking: we want to include this?

Reply:
oops thx, i didn't notice that i had committed that.